### PR TITLE
CBBF-106: FHIR Mapping Change (Carrier)

### PR DESCRIPTION
**Pull request by jcgeer**
- Referral Request recipient remapped to PFR_PHYSN_NPI/PRVDR_NPI for
Carrier/DME respectively.
- Carrier LPRPAYCD extension URL and system value cahnged to point to
LPRPAYCD.txt

### CODE REVIEW

File: bluebutton-server-app/src/main/java/gov/hhs/cms/bluebutton/server/app/stu3/providers/TransformerConstants.java [hunk 1]
```diff
@@ -469,6 +469,8 @@
 
 	public static final String EXTENSION_CODING_PRIMARY_PAYER = "https://www.ccwdata.org/cs/groups/public/documents/datadictionary/prpay_cd.txt";
 
+	public static final String EXTENSION_CODING_CARRIER_PRIMARY_PAYER = "https://www.ccwdata.org/cs/groups/public/documents/datadictionary/LPRPAYCD.txt";
```
Comment (glovercloud) [inline bluebutton-server-app/src/main/java/gov/hhs/cms/bluebutton/server/app/stu3/providers/TransformerConstants.java]: Should the text file name be lowercase to match the others?

**Comment by dho937** [comment]
@jcgeer  Can you fix this conflict and then merge your code into master?  Thanks.

**Comment by dho937** [comment]
@jcgeer   Sorry...is this still waiting on Susie's reply?  Thanks.

**Comment by jcgeer** [comment]
Waiting on Susie's reply for one last thing, but I think it should be good to go otherwise. I've resolved the conflicts.

**Pull request merged**